lib/widget/dropdown.dart

```diff
@@ -199,9 +209,31 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
     if (items != null) {
       results = [];
       for (SelectOneItem item in items) {
-        results.add(DropdownMenuItem(
-          value: item.value,
-          child: Text(Utils.optionalString(item.label) ?? item.value)));
+        item.iconname != null
```

Comment:
if a row has icon and another doesn't, they won't align properly. Can we keep the spacing if at least one item has icon?

Reply:
yes we can